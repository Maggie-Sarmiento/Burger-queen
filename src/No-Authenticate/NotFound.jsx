import React from "react";

const NotFound = () => {
  return (
    <img src="https://svgshare.com/i/gnR.svg" alt="img404" />
  );
}

export default NotFound;