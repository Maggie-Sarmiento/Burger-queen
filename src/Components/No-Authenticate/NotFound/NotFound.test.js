/* eslint-disable react/jsx-filename-extension */
/* eslint-disable no-undef */
// import { render, screen } from '@testing-library/react';
import React from 'react';
import { create, act } from 'react-test-renderer';
import NotFound from './NotFound';

describe('test notfound', () => {
  test('render Notfound', async () => {
    let root;
    act(() => {
      root = create(
        <NotFound />,
      );
    });
    expect(root.toJSON()).toMatchSnapshot();
  });
});
