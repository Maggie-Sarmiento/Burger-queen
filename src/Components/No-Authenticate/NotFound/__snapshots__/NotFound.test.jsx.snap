// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test notfound render component Notfound 1`] = `
<img
  alt="img404"
  src="https://svgshare.com/i/gnR.svg"
/>
`;
